doc/field_guide__variable.md

```diff
@@ -0,0 +1,27 @@
+## Variable stars (variable)
+Variable stars are objects whose brightness is varying at a statistically significant level.

```

Comment:
Some bla-bla about how survey-specific this thus is?

Reply:
Yes indeed too wishy washy.